flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -1195,12 +1204,11 @@ public void internalCopySerializerTest(
                         previousSerializer.snapshotConfiguration());
         testProvider.registerNewSerializerForRestoredState(newSerializerForRestoredState);
 
-        assertEquals(
-                testProvider.currentSchemaSerializer().getClass(),
-                internalCopySerializer.getClass());
+        assertThat(internalCopySerializer.getClass())
+                .isEqualTo(testProvider.currentSchemaSerializer().getClass());
     }
 
-    @Test
+    @TestTemplate
     public void testStateMigrationAfterChangingTTL() throws Exception {
```

Comment:
```suggestion
    void testStateMigrationAfterChangingTTL() throws Exception {
```
could be package private